src/Wallabag/CoreBundle/Resources/views/themes/common/Entry/share.html.twig

```diff
@@ -27,11 +27,18 @@
                 width: 600px;
             }
         </style>
+        <meta property="og:title" content="{{ entry.title | raw }}" />
+        <meta property="og:type" content="article" />
+        <meta property="og:url" content="{{ app.request.uri }}" />
+        {% if entry.previewPicture is not null %}
```

Comment:
@nicosomb Maybe if the previewPicture is null we could have displayed wallabag's logo ?